README.md

```diff
@@ -10,7 +10,59 @@ Currently this project has the following functionality:
 
 ## How to use this?
 
-Instructions coming soon :)
+### Installation
+
+```
+Note: The following will only work after we release the first version of the extension.
+```
+
+Prerequisites:
+
+- Running [Rancher Manager cluster](https://ranchermanager.docs.rancher.com/) with cert-manager
+- [Helm](https://helm.sh/)
+
+Quick start:
+
+These commands will install: Rancher turtles extension, CAPI Operator, and the core CAPI CRDs and controllers.
+
+```bash
+helm repo add rancher-turtles https://rancher-sandbox.github.io/rancher-turtles
+helm repo update
+helm install rancher-turtles rancher-turtles/rancher-turtles --create-namespace -n rancher-turtles-system
+```
+
+Customizing the deployment:
+
+The Rancher turtles Helm chart supports the following values:
+
+```yaml
+rancherTurtles:
+  image: # image to use for the extension 
+  tag: # tag to use for the extension
+  imagePullPolicy: # image pull policy to use for the extension
+  namespace: # namespace to deploy to (default: rancher-turtles-system)
+capi:
+  enabled: # indicates if core CAPI controllers should be installed (default: true)
+  namespace: # namespace to create and deploy core CAPI controllers (default: capi-system)
+  configSecret:
+    name: # name of the config secret to use for core CAPI controllers, used by the CAPI operator. See [CAPI operator](https://github.com/kubernetes-sigs/cluster-api-operator/tree/main/docs#installing-azure-infrastructure-provider) docs for more details.
+    namespace: # namespace of the config secret to use for core CAPI controllers, used by the CAPI operator.
+  fetchConfig: # (only required for airgapped environments)
+    url: # url to fetch config from, used by the CAPI operator. See [CAPI operator](https://github.com/kubernetes-sigs/cluster-api-operator/tree/main/docs#provider-spec) docs for more details.
+    selector: # selector to use for fetching config, used by the CAPI operator.
+cluster-api-operator:
+  enabled: # indicates if CAPI operator should be installed (default: true)
+```
+### Installing CAPI providers
+
+The Rancher turtles extension does not install any CAPI providers, you will need to install them yourself using [CAPI operator](https://github.com/kubernetes-sigs/cluster-api-operator/tree/main/docs).
```

Comment:
But you can, using infrastructure flag passed to helm values, no?

Reply:
Not really, in this case, CAPI operator chart will deploy the core CAPI provider too. We tried to follow the same pattern as clusterctl here, our deployment model is a bit more complicated. What can help here is a clusterctl plugin for operator or alternatively rancher users will get a UI for providers